src/frecuencies.ts

```diff
@@ -0,0 +1,16 @@
+import { IFrecuency } from './interfaces'
+
+export const FRECUENCIES: IFrecuency[] = [
+  {
+    name: 'Month',
+    code: 'M',
```

Comment:
make code human readable, like month or week or live. No need to create single letter code
```suggestion
    code: 'month',
```